src/icons/general/IconSecurityClassificationSolid.js

```diff
@@ -3,44 +3,38 @@ import * as React from 'react';
 import classNames from 'classnames';
 
 import AccessibleSVG from '../accessible-svg';
-import { bdlGray, white } from '../../styles/variables';
+import { bdlGray, bdlGray20 } from '../../styles/variables';
 
 type Props = {
     className?: string,
     color?: string,
+    fillColor?: string,
     height?: number,
-    strokeWidth?: number,
+    strokeColor?: string,
     /** A text-only string describing the icon if it's not purely decorative for accessibility */
     title?: string | React.Element<any>,
     width?: number,
 };
 
 const IconSecurityClassificationSolid = ({
     className = '',
-    height = 32,
-    color = bdlGray,
+    fillColor = bdlGray20,
+    height = 13,
+    strokeColor = bdlGray,
     title,
-    strokeWidth = 2,
-    width = 32,
+    width = 13,
 }: Props) => {
     const classes = classNames('bdl-IconSecurityClassificationSolid', className);
 
     return (
-        <AccessibleSVG className={classes} height={height} title={title} viewBox="0 0 32 32" width={width}>
-            <path
-                d="M17,2 L5,8 L5,15 C5,21.4214876 10.6933333,29.5421488 17,31 C23.3066667,29.5421488 29,21.4214876 29,15 L29,8 L17,2 Z"
-                stroke={color}
-                strokeWidth={strokeWidth}
-                fill={color}
-            />
-            <path
-                d="M23,11 L23,19"
-                stroke={white}
-                strokeWidth={strokeWidth}
-                strokeLinecap="round"
-                strokeLinejoin="round"
-                fill="none"
-            />
+        <AccessibleSVG className={classes} height={height} title={title} viewBox="0 0 10 13" width={width}>
```

Comment:
The icon size should be 32 x 32. I am not sure if we have something small but should follow the bdl standard.

Reply:
We have 32 x 32, 16 x 16, and 20 x 20

Reply:
Chat in person, JP will make another PR once got reply from design.

Reply:
Icon updated